sage_modabvar/abvar.py

```diff
@@ -2412,6 +2412,94 @@ def frobenius_polynomial(self, p, var='x'):
 
         return ZZpoly(ans.univariate_polynomial())
 
+    def all_twists(self):
```

Comment:
This name is misleading, since there are infinitely many twists that are not inner twists.